content/tools/figma.mdx

```diff
@@ -19,12 +19,11 @@ Design teams at GitHub use Figma as the single source of truth, collaboration, a
 
 ## Team libraries
 
-GitHub uses Figma to distribute Primer through team libraries. Each library covers a different part of the system, allowing a designer to only use what's needed. Any team within GitHub's Figma organization can create and distribute their own team/project specific libraries, but all of the core asset libraries for Primer can be found in the Primer team's "[Core libraries](https://www.figma.com/files/797266083973274396/project/8172024/Core-libraries)" project. They include the following:
+GitHub uses Figma to distribute Primer through team libraries. Each library covers a different part of the system, allowing a designer to only use what's needed. Any team within GitHub's Figma organization can create and distribute their own team/project specific libraries, but all of the core asset libraries for Primer can be found in the Primer team's "[Core libraries](https://www.figma.com/@primer)" project. They include the following:
 
-- [Primer Primitives](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives): All of the color, type, and spacing styles used within Primer.
-- [Primer Web](https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web): UI components used to design for GitHub's web interfaces.
-- [Primer Mobile](https://www.figma.com/file/csCViryWkkXwBsK1aLrO0X/Primer-Mobile): UI components used to design for GitHub's native mobile platforms.
-- [Octicons](https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons): GitHub's custom icon library.
+- [Primer Primitives](https://www.figma.com/community/file/854766928300977832): All of the color, type, and spacing styles used within Primer.
```

Comment:
```suggestion
- Primer Primitives ([Internal](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives) | [Community](https://www.figma.com/community/file/854766928300977832)): All of the color, type, and spacing styles used within Primer.
```

You could change it to have an Internal and a community link for the items. Mobile is internal only at the moment.